src/ontology/foodon-edit.owl

```diff
@@ -18460,6 +18460,125 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
     
 
 
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003928 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003928">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03310775"/>
+        <obo:IAO_0000115 xml:lang="en">A pizza food product which is an assembled pizza including pizza crust, pizza sauce and toppings.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-06T12:23:03Z</dc:date>
+        <rdfs:label xml:lang="en">pizza</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003929 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003929">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003928"/>
+        <obo:IAO_0000115 xml:lang="en">A pizza made with tomatoes, mozzarella cheese, fresh basil, salt, and extra-virgin olive oil.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://en.wikipedia.org/wiki/Pizza_Margherita</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-09T12:53:11Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">margherita pizza</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">pizza (margherita)</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003930 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003930">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003928"/>
+        <obo:IAO_0000115 xml:lang="en">A pizza made with pineapple and either ham or bacon as toppings.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2000/01/rdf-schema#anyURI">https://en.wikipedia.org/wiki/Hawaiian_pizza</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-09T13:15:27Z</dc:date>
+        <rdfs:label xml:lang="en">hawaiian pizza</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003931 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003931">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003928"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03311484"/>
+        <obo:IAO_0000115 xml:lang="en">A pizza made with barbeque sauce, diced barbecue chicken, cilantro, peppers, and onion all covered with cheese.</obo:IAO_0000115>
```

Comment:
Also, cilantro required?  Be careful to list only the ingredients that are absolutely essential in the definition.